jenkins/Jenkinsfile-external

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+@Library('h2o3-shared-lib') _
+// Job parameters
+properties(
+        [
+                parameters(
+                        [
```

Comment:
The jobs properties are duplicated - that makes maintenance harder.

Reply:
I know.., they are however a bit different - we don't need to set properties related to external cluster in internal jenkinsfile and vice versa.

We could extract the shared configuration to some shared repo with Jenkins functions, however that seemed  too much for this ( at least to me )